.vscode/tasks.json

```diff
@@ -90,198 +26,110 @@
 			"type": "shell",
-			"command": "velocitas exec runtime-k3d configure_controlplane",
+			"command": "velocitas exec runtime-local run-vehicledatabroker-cli",
 			"group": "none",
-			"presentation": {
-				"reveal": "always",
-				"clear": true,
-				"close": false
-			},
+			"isBackground": false,
 			"problemMatcher": [ ]
 		},
 		{
-			"label": "K3D - Deploy runtime",
-			"detail": "Deploys the runtime components (like Vehicle Data Broker, Seat Service, ...) within the K3D cluster.",
+			"label": "Pre Commit Action",
+			"detail": "Perform checking/formating of the code",
 			"type": "shell",
-			"command": "velocitas exec runtime-k3d deploy_runtime",
+			"command": "pre-commit run --show-diff-on-failure --color=always --all-files",
 			"group": "none",
-			"presentation": {
-				"reveal": "always",
-				"clear": true,
-				"close": false
-			},
-			"dependsOn": [
-				"K3D - Configure control plane"
-			],
+			"isBackground": false,
 			"problemMatcher": [ ]
 		},
 		{
-			"label": "K3D - Build VehicleApp",
-			"detail": "Builds the VehicleApp and pushes it to the local K3D registry.",
+			"label": "K3D Runtime - Up",
+			"detail": "Starts up the K3D runtime.",
 			"type": "shell",
-			"command": "velocitas exec deployment-k3d build_vehicleapp",
+			"command": "velocitas exec runtime-k3d up",
 			"group": "none",
 			"presentation": {
 				"reveal": "always",
-				"panel": "dedicated",
 				"clear": true,
 				"close": false
 			},
 			"problemMatcher": [ ]
 		},
 		{
-			"label": "K3D - Deploy VehicleApp",
-			"detail": "Builds and deploys the VehicleApp via Helm to the K3D cluster.",
+			"label": "K3D Runtime - Down",
+			"detail": "Stops up the local runtime",
 			"type": "shell",
-			"command": "velocitas exec deployment-k3d deploy_vehicleapp",
+			"command": "velocitas exec runtime-k3d down",
 			"group": "none",
 			"presentation": {
 				"reveal": "always",
-				"panel": "dedicated",
 				"clear": true,
 				"close": false
 			},
-			"dependsOn": [
-				"K3D - Deploy runtime",
-				"K3D - Build VehicleApp"
-			],
 			"problemMatcher": [ ]
 		},
 		{
-			"label": "K3D - Deploy VehicleApp (without rebuild)",
-			"detail": "Deploys the VehicleApp via Helm to the K3D cluster (without rebuilding it)",
+			"label": "K3D Runtime - Build VehicleApp",
+			"detail": "Builds the VehicleApp and pushes it to the local K3D registry.",
 			"type": "shell",
-			"command": "velocitas exec deployment-k3d deploy_vehicleapp",
+			"command": "velocitas exec deployment-k3d build-vehicleapp",
 			"group": "none",
 			"presentation": {
 				"reveal": "always",
 				"panel": "dedicated",
 				"clear": true,
 				"close": false
 			},
-			"dependsOn": [
-				"K3D - Deploy runtime"
-			],
 			"problemMatcher": [ ]
 		},
 		{
-			"label": "K3D - Uninstall runtime",
-			"detail": "Uninstalls the runtime components from the K3D cluster (without deleting the cluster).",
+			"label": "K3D Runtime - Deploy VehicleApp",
```

Comment:
Deployed application was not installed on k3d. Task ends with error.

Reply:
```
The push refers to repository [localhost:12345/sampleapp]
5f70bf18a086: Preparing
e8d2951bcc08: Preparing
d7503d339da2: Preparing
5f70bf18a086: Layer already exists
e8d2951bcc08: Pushed
d7503d339da2: Pushed
local: digest: sha256:5c1796ea58083392d11d7a1361665590f84e35ef379437ebd14b4c32c225b4c4 size: 941
NAME: vapp-chart
LAST DEPLOYED: Thu May 25 17:03:27 2023
NAMESPACE: default
STATUS: failed
REVISION: 1
TEST SUITE: None
release "vapp-chart" uninstalled
install.go:200: [debug] Original chart version: ""
install.go:217: [debug] CHART PATH: /home/vscode/.velocitas/packages/devenv-runtimes/v1.0.0/runtime-k3d/src/app_deployment/config/helm

client.go:134: [debug] creating 1 resource(s)
wait.go:48: [debug] beginning wait for 1 resources with timeout of 1m0s
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
ready.go:277: [debug] Deployment is not ready: default/sampleapp. 0 out of 1 expected pods are ready
Error: INSTALLATION FAILED: context deadline exceeded
helm.go:84: [debug] context deadline exceeded
INSTALLATION FAILED
main.newInstallCmd.func2
	helm.sh/helm/v3/cmd/helm/install.go:147
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.6.1/command.go:916
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.6.1/command.go:1044
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.6.1/command.go:968
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:250
runtime.goexit
	runtime/asm_amd64.s:1598
Command '['helm', 'install', 'vapp-chart', '/home/vscode/.velocitas/packages/devenv-runtimes/v1.0.0/runtime-k3d/src/app_deployment/config/helm', '--values', '/home/vscode/.velocitas/packages/devenv-runtimes/v1.0.0/runtime-k3d/src/app_deployment/config/helm/values.yaml', '--set', 'imageVehicleApp.repository=k3d-registry.localhost:12345/sampleapp', '--set', 'imageVehicleApp.name=sampleapp', '--set', 'imageVehicleApp.daprAppid=sampleapp', '--set', 'imageVehicleApp.daprPort=50008', '--wait', '--timeout', '60s', '--debug']' returned non-zero exit status 1.
```

Reply:
how does the cpp app behave on "old template" what could be different to the new approach?